File: Practical Programming/Cola Machine/Python/Cola Machine/Cola_Machine.py
from time import sleep

print("Hello and Welcome to this non-existent Vending Machine!")

sleep(2)

print("Choose your beverage!\n")

print("1. Pepsi")
print("2. Coke")
print("3. Sprite")
print("4. 7-up")
print("5. Milk")

choice = int(input())

if choice == 1:
    print("You chose Pepsi!")
    sleep(2)

elif choice == 2:
    print("You chose Coke!")
    sleep(2)

elif choice == 3:
    print("You chose Sprite!")
    sleep(2)

elif choice == 4:
    print("You chose 7-up!")
    sleep(2)

elif choice == 5:
    print("You chose Milk!")
    sleep(2)

else:
    print("Hey, that was not vaild!")
    sleep(1)
    print("Here is your money back!")

File: PracticalCplusplusAndPythonTutorial/Cola Machine/C++/Cola Machine.cpp
// Cola Machine.cpp : Defines the entry point for the console application.
//

#include "stdafx.h"

using namespace std;

int main()
{
	cout << "Hello and Welcome to this non-existent Vending Machine!" << endl;

	this_thread::sleep_for(2s);

	cout << "Choose your Beverage!\n" << endl;
	cout << "1. Water" << endl;
	cout << "2. Sparking Water" << endl;
	cout << "3. Lemon Sparkling Water" << endl;
	cout << "4. Orange Sparkling Water" << endl;
	cout << "5. Milk" << endl;

	int choice = 0;

	cin >> choice;

	switch (choice) {
	case 1:
			cout << "You chose Water!" << endl;
			system("PAUSE");
			break;
	case 2:
		cout << "You chose Sparking Water" << endl;
		system("PAUSE");
		break;

	case 3:
		cout << "You chose Lemon Sparking Water" << endl;
		system("PAUSE");
		break;

	case 4:
		cout << "You chose Orange Sparking Water" << endl;
		system("PAUSE");
		break;

	case 5:
		cout << "You chose Milk" << endl;
		system("PAUSE");
		break;

	default:
		cout << "Hey, that was not vaild!" << endl;
		this_thread::sleep_for(1s);
		cout << "Here is a refund!" << endl;
		system("PAUSE");

	}

    return 0;
}

